### PR TITLE
sysfs: split out topology to a subdir (and subpackage) of its own.

### PR DESCRIPTION
**Pull request by klihub**
Split out topology from sysfs for low overhead importing to other packages.

### CODE REVIEW

**Comment by mythi** [comment]
@bart0sh

File: go.mod [hunk 1]
```diff
@@ -28,6 +28,7 @@ require (
 
 // versions are based on k8s.io/kubernetes
 replace (
+	github.com/intel/cri-resource-manager/pkg/topology v0.0.0 => ./pkg/topology
```
Comment (marquiz) [inline go.mod]: Umm, what is this for?

Comment (klihub) [reply go.mod]: For being able to build cri-resmgr at all in CI. Without this go mod looks for it in a git repository at github.com/intel/cri-resource-manager/pkg/topology which then fails.

**Comment by mythi** [comment]
rebase?

**Comment by marquiz** [comment]
Code looks good. Just add an explanation why this is done in the commit message

**Comment by mythi** [comment]
@klihub thanks!

**Pull request merged**